__tests__/app.test.js

```diff
@@ -177,3 +178,39 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(Array.isArray(body.articles)).toBe(true);
+      });
+  });
+  it("should return the articles sorted in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(body.articles).toBeSorted({
+          key: "created_at",
+          descending: true,
+        });
+      });
+  });
+  it("should return the articles with the comment_count key", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(body.articles.length).not.toBe(0);
+        body.articles.forEach((article) => {
+          expect(article.hasOwnProperty("comment_count")).toBe(true);
+        });
+      });
+  });
+});
```

Comment:
Good stuff Aaron - you'll need to add a test here to check that you're not getting the body back as it should not be included - again I'd recommend using (objectIncluding) for this.

Reply:
could you clarify what you mean here please?